app/uk/gov/hmrc/tariffclassificationfrontend/views/forms/error_summary.scala.html

```diff
@@ -0,0 +1,31 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(form: Form[_])(implicit messages: Messages)
+
+@if(form.hasErrors) {
+ <div id="sla_criteria-errors" class="error-summary" role="alert" aria-labelledby="sla_criteria-error" tabindex="-1">
```

Comment:
Is this component specific to errors on SLA reports?   If so, should the file name reflect this?

Reply:
Nah it's copied from the trader UI